src/test/java/jenkins/plugins/git/GitSCMSourceTest.java

```diff
@@ -367,6 +365,13 @@ public void gitSCMSourceShouldResolveToolsForMaster() throws Exception {
         assertTrue("Installer should be invoked", inst.isInvoked());
     }
 
+    @Test
+    public void checkCredentialIdFormValidationOk () throws Exception{
+        GitSCMSource.DescriptorImpl scmSource = new GitSCMSource.DescriptorImpl();
+        FormValidation validation = scmSource.doCheckCredentialsId(null,"https://github.com/jenkinsci/git-plugin","");
+        Assert.assertEquals(validation,FormValidation.ok());
+    }
+
```

Comment:
Thanks for adding the test.

Please add a test of the negative case as well.  

You're welcome to use 'https://github.com/jenkinsci-cert/git-plugin.git` and `git@github.com:jenkinsci-cert/git-plugin.git` as repositories which exist but are not readable to others.  Alternately, you can refer to a repository that does not exist (since the response is likely the same) as in https://github.com/jenkinsci/no-such-repo.git

Reply:
Since the first two of those repositories can be read by me (and hardly anyone else), I'll likely extend the test to use a different repository if the test detects that it is running as me.  You can write the test as yourself, since you don't have access to the jenkinsci-cert repository, and then I can adapt it (if needed) to run in my environment.